15_tictactoe/go/game/game.go

```diff
@@ -29,61 +29,39 @@ Check if there is a winner, if yes, exit, if no, continue from the start
 func Play() {
 	// Initialize empty grid
 	g := grid.EmptyGrid()
-	fmt.Println(g.String())
-
-	for i := 0; i <= 9; i++ {
-		// Output empty grid to help understand the concept?
+	i := 0
+	for {
```

Comment:
You could put the initialization of i in the `for` line and skip the break condition:
```suggestion
	for i := 0 ;; i++{
```

Reply:
Amazing, thank you!
I was wondering if I could use this syntax and not set a break condition initially.
Done!